src/modules/catalog.js

```diff
@@ -1584,6 +1584,109 @@ class Catalog {
     }
   }
 
+  /**
+   * Send patch delta catalog files to patch the current catalog
+   *
+   * @function patchCatalog
+   * @param {object} parameters - Additional parameters for catalog details
+   * @param {string} parameters.section - The section that you want to update
+   * @param {string} [parameters.notification_email] - An email address where you'd like to receive an email notifcation in case the task fails.
+   * @param {boolean} [parameters.force] - Process the catalog even if it will invalidate a large number of existing items. Defaults to False.
```

Comment:
We can specify defaults using JSDoc: https://jsdoc.app/tags-param.html#optional-parameters-and-default-values

Reply:
Sounds good. Also added the default for `force` under `replaceCatalog` and `updateCatalog`